pydicom/values.py

```diff
@@ -467,18 +537,23 @@ def convert_single_string(byte_string, encodings=None):
     """
     encodings = encodings or [default_encoding]
     value = decode_string(byte_string, encodings, TEXT_VR_DELIMS)
-    while value and (value.endswith(' ') or value.endswith('\0')):
+    while value and value.endswith((' ', '\0')):
         value = value[:-1]
     return value
```

Comment:
Can we not just change to `rstrip("\0 ")` here (and elsewhere)? -- simpler and faster than a loop.

Reply:
Will do